libs/execution_context/ebpf_link.c

```diff
@@ -652,7 +652,7 @@ ebpf_link_get_info(
 
     memset(info, 0, sizeof(*info));
     info->id = link->object.id;
-    info->prog_id = (link->program) ? ((ebpf_core_object_t*)link->program)->id : EBPF_ID_NONE;
+    info->prog_id = (link->program) ? ((ebpf_core_object_t*)link->program)->id : 0;
```

Comment:
Question: should EBPF_ID_NONE be defined to be 0?

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/4098